packages/botonic-react/src/components/whatsapp-product-list.tsx

```diff
@@ -0,0 +1,55 @@
+import { INPUT } from '@botonic/core'
+import React from 'react'
+
+import { renderComponent } from '../util/react'
+import { Message } from './message'
+
+export interface ProductItem {
+  product_retailer_id: string
+}
+
+export interface WhatsappProductListSection {
+  title: string
+  product_items: ProductItem[]
```

Comment:
for consistency with the api you are defining, I would put productItems instead of product_items and later do the conversion to what backend will use.